jupyterhub/handlers/pages.py

```diff
@@ -67,9 +67,6 @@ class HomeHandler(BaseHandler):
         else:
             url = url_path_join(self.hub.base_url, 'spawn', user.escaped_name)
 
-        auth_state = await user.get_auth_state()
-        user.spawner.run_auth_state_hook(auth_state)
-
```

Comment:
Removing this will make us unable to use a users auth_state in templates to render /hub/home, right?

Should we perhaps have this remain here as well? Here is a render of /hub/home from my jupyterhub for example, and it could make sense in my mind to write out something one has stored in auth_state here.

![image](https://user-images.githubusercontent.com/3837114/72423431-8a140500-3784-11ea-96dd-388c0b1d7131.png)

Reply:
I don't think so. Only the *spawner* shouldn't be able to take actions based on the auth state, which shouldn't happen on the home page, I think.

Reply:
Is there another/right way to get the auth state information to the template?